sdk/go/core/docsgen/guides/presence_class.rst

```diff
@@ -22,10 +22,25 @@ According to `RFC 6020 <https://tools.ietf.org/html/rfc6020#section-7.5.1>`_, YA
         PrefixListEntries Ipv4AclAndPrefixList_Prefixes_Prefix_PrefixListEntries
     }
 
+    // Ipv4AclAndPrefixList_Prefixes_Prefix_PrefixListEntries
+    // Sequence of entries forming a prefix list
+    // This type is a presence type.
+    type Ipv4AclAndPrefixList_Prefixes_Prefix_PrefixListEntries struct {
+        EntityData types.CommonEntityData
+        YFilter yfilter.YFilter
+        YPresence bool
+
+        // A prefix list entry; either a description (remark) or a prefix to match
+        // against. The type is slice of
+        // Ipv4AclAndPrefixList_Prefixes_Prefix_PrefixListEntries_PrefixListEntry.
+        PrefixListEntry []Ipv4AclAndPrefixList_Prefixes_Prefix_PrefixListEntries_PrefixListEntry
+}
+
 Since the existence of container ``prefix-list-entries`` itself represents configuration data, YDK does not instantiate an instance of class :go:struct:`Ipv4AclAndPrefixList_Prefixes_Prefix_PrefixListEntries<ydk/models/cisco_ios_xr/ipv4_acl_cfg/Ipv4AclAndPrefixList/Prefixes/Prefix/PrefixListEntries>` and assign it to the ``PrefixListEntries`` leaf. The user needs to manually instantiate and assign this object as follows:
```

Comment:
Suggest changing to something like:


Since the existence of container ``prefix-list-entries`` itself represents configuration data, YDK marks the instance of class:go:struct:`Ipv4AclAndPrefixList_Prefixes_Prefix_PrefixListEntries<ydk/models/cisco_ios_xr/ipv4_acl_cfg/Ipv4AclAndPrefixList/Prefixes/Prefix/PrefixListEntries>` (``PrefixListEntries``) leaf with the `YPresence` attribute . The user needs to manually set this attribute as follows: